queries.sql

```diff
@@ -9,3 +9,50 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg <= 17.3 AND weight_kg >= 10.4;
 
+/* transactions */
+BEGIN;
+UPDATE animals
+SET species = 'unspecified'
+WHERE species IS null;
+COMMIT;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.), Please make sure that you change all species to `unspecified`, not only the null ones and kindly verify that the change is done then rollback and verify again that change was undone.